submissions/ZjeniiaSt/html-css-popup/styles.css

```diff
@@ -0,0 +1,172 @@
+:root {
+  --background-color-block: rgba(245, 244, 250, 1);
+  --white-color: rgba(255, 255, 255, 1);
+  --main-font: "Roboto", sans-serif;
+  --timing: cubic-bezier(0.4, 0, 0.2, 1);
+}
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+/*geometry*/
+html {
+  box-sizing: border-box;
+}
+
+*,
+*::before,
+*::after {
+  box-sizing: inherit;
+}
+
+body {
+  background-color: var(--white-color);
+  font-family: var(--main-font);
+  font-style: normal;
+  letter-spacing: 0.03em;
+  margin: 0;
+}
+
+a {
+  color: inherit;
+  text-decoration: none;
+  cursor: pointer;
+}
+
+ul {
+  list-style: none;
+}
+
+img {
+  display: block;
+  width: 100%;
+  height: auto;
+}
+
+.container {
+  margin: 0 auto;
+  padding-left: 15px;
+  padding-right: 15px;
+  width: 100%;
+}
+
+.wrapper {
+  display: flex;
+  justify-content: end;
+  flex-direction: column;
+  margin: 30px;
+  padding: 15px;
+}
+
+.nav {
+  display: flex;
+  text-align: center;
+  justify-content: end;
+}
+
+.nav-item {
+  cursor: pointer;
+  padding: 5px;
+}
+
+.nav-item:not(:last-child) {
+  margin-right: 15px;
+}
+label {
+  cursor: pointer;
+}
+
+input {
+  display: none;
```

Comment:
[Watch video](https://www.youtube.com/watch?v=E6kLaaQFctU&t=413s) to find out how to hide checkbox in the right way.